config/steps/composer.yml

```diff
@@ -9,8 +9,13 @@
   register: composer_self_update_result
   changed_when: composer_self_update_result.stderr is search('Updating')
 
-- name: Install composer
+- name: Install stable composer
   get_url: url=https://getcomposer.org/composer-stable.phar dest={{symfony_composer_path}} mode=0755 validate_certs=no force=no
+  when: symfony_composer_version is undefined
```

Comment:
I would use `is not defined` here because it's more readable.

Reply:
I made the change. The ansible doc is not really clear on the difference between `is not definied` and `is undefined`